lib/cssLoadChecker/README.md

```diff
@@ -0,0 +1,19 @@
+# CSS Load Checker
+
+This module provides a means of checking if a CSS file has been loaded already or not. It does this by adding a test div to the page and checking the computed styles of it for the `display: none` style.
+
+```js
+var cssLoadChecker = require('bv-ui-core/lib/cssLoadChecker');
+
+if (cssLoadChecker.checkForCss('bv-verify-css-loaded')) {
```

Comment:
1. Should this be a promise? Technically the promise only needs to be resolved once as once css loads, it's "loaded" and can't be unloaded. Am I just overengineering? @savetheclocktower? @tnunamak?

```javascript
cssLoadChecker.cssIsLoaded('bv-verify-css-loaded')
  .then(() => { 
    // Continue with your life.
  }
  .fail(() => {
     loader.loadStyleSheet(namespace.cssUrl, callback)
  })
```

2. This is personal preference: `checkForCss` -> `isCssLoaded` since it returns a boolean.